### ISSUE TITLE
Improve the transfer screen

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#1460]
## Description
After #1212 the transfer screen looks quite cluttered. We should go the next step to improve it.

Maybe building on top of the ideas we have collected/discussed within #571 and #586 

## Acceptance criteria
- Looks nicer

## Tasks
- [ ]

**Comment by christianbrb** [comment]
Maybe going into the direction of the card design https://github.com/raiden-network/light-client/issues/571#issuecomment-557081967 or aligning with the new WebUI design.

**Comment by christianbrb** [comment]
Meeting with @sashseurat and @taleldayekh 
1. Improving the transfer area
2. Changing the menu (top screen)
- Going with the WebUI Redesign
- ![grafik](https://user-images.githubusercontent.com/35398162/84642977-83d5b080-aefd-11ea-867d-0ac837a2b66c.png)
3. Removing the address bar

**Comment by sashseurat** [comment]
Here is a click-dummy draft ready for review:
https://www.figma.com/file/3UX5wudtGolsdh5QVyuoTS/LC-Header?node-id=0%3A1

Let me know when you'd like to have a talk about it, to explain some details.

**Comment by christianbrb** [comment]
@sashseurat Great, thanks :) Looks nice!

Some smaller remarks:
- The turquoise looks different. Maybe it is due to the fact, that the font color is typically white
- I would out scope the channel integration on the screen for now. This might be too much work close to main net. Could you add it to the menu. For future implementation I have created #1716. 
- I am not sure about the net (Goerli) being very prominent on the top of the screen. 
- I like the deposit icons we have used before.
- I am not sure about the bell (notification icon).

@taleldayekh What are your thoughts?

**Comment by christianbrb** [comment]
@sashseurat I just had a brief discussion with @taleldayekh 

In the first iteration of the improvement, we would like to focus on the token and transfer part of the screen. I am going to create a separate issue to change the header in the future.

@taleldayekh is going to approach @sashseurat to finetune the details of the two parts.

![grafik](https://user-images.githubusercontent.com/35398162/84876509-94b22d80-b087-11ea-8996-c88d29df0eac.png)

**Comment by christianbrb** [comment]
Issue to change the header of the dApp -> #1717

**Comment by kelsos** [comment]
> @sashseurat I just had a brief discussion with @taleldayekh
> 
> In the first iteration of the improvement, we would like to focus on the token and transfer part of the screen. I am going to create a separate issue to change the header in the future.
> 
> @taleldayekh is going to approach @sashseurat to finetune the details of the two parts.
> 
> ![grafik](https://user-images.githubusercontent.com/35398162/84876509-94b22d80-b087-11ea-8996-c88d29df0eac.png)

That looks really nice :D

**Comment by sashseurat** [comment]
> @sashseurat Great, thanks :) Looks nice!
> 
> Some smaller remarks:
> 
> * The turquoise looks different. Maybe it is due to the fact, that the font color is typically white
> * I would out scope the channel integration on the screen for now. This might be too much work close to main net. Could you add it to the menu. For future implementation I have created #1716.
> * I am not sure about the net (Goerli) being very prominent on the top of the screen.
> * I like the deposit icons we have used before.
> * I am not sure about the bell (notification icon).

The icons I used are placeholders for whatever icons are in the light client, as it doesn't make sense to spend the time on importing the exact icons that exist in the light client into Figma :)

**Comment by sashseurat** [comment]
FYI:
The "Please use up to ...." text is conditional and should only be displayed if user inputs wrongly.

**Comment by sashseurat** [comment]
Also the prototype shows the old Transfer component structure if you drag it :)
